include/SFML/Window/Joystick.hpp

```diff
@@ -88,7 +85,7 @@ class SFML_WINDOW_API Joystick
     /// \return True if the joystick is connected, false otherwise
     ///
     ////////////////////////////////////////////////////////////
-    static bool isConnected(unsigned int joystick);
+    bool isConnected(unsigned int joystick);
```

Comment:
I think you'll need to mark all these functions with `SFML_WINDOW_API` to fix shared library linking issues.

Reply:
That would make sense. I was trying to figure out what I needed to do. Thank you!